icu4c/source/i18n/gregocal.cpp

```diff
@@ -324,26 +324,28 @@ GregorianCalendar::setGregorianChange(UDate date, UErrorCode& status)
     if (U_FAILURE(status)) 
         return;
 
-    fGregorianCutover = date;
+    fGregorianCutover = (int64_t)date;
```

Comment:
Why? It gets pinned below if necessary.
I assume that the field is a UDate=double.

Reply:
fixed. (actually this line was a bad merge, line 348 is identical in the normal case)